v4/source/users.yaml

```
@@ -346,6 +393,15 @@
           $ref: '#/responses/Forbidden'
         '404':
           $ref: '#/responses/NotFound'
+      x-code-samples:
+        - lang: 'Go'
+          source: |
+            import "github.com/mattermost/mattermost-server/model"
+
+            Client := model.NewAPIv4Client("https://your-mattermost-url.com")
+            Client.Login("email@domain.com", "Password1")
+
+            stats, resp := Client.GetTotalUsersStats("").TotalUsersCount
```

I would like to stay consistent here and just get the response.
```suggestion
            stats, resp := Client.GetTotalUsersStats("")
```

---

done